packages/smooth_app/lib/data_models/user_preferences_model.dart

```diff
@@ -149,7 +150,8 @@ class PreferencesVariable {
   final String description;
   final String descriptionShort;
 
-  static const String VEGAN = 'vegan';
+  @deprecated
+  static const String VEGAN = 'vegan'; // TODO(monsieurtanuki): still relevant?
```

Comment:
We probably will not use the attribute id strings in Smooth App (as for Smooth App we will use all attributes returned by the attribute_groups API), I guess they could be useful for other apps. For instance an app for vegans that specifically looks for the vegan attribute.